<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>793</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Modbus串口读写工具 v1.0</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="autoFillBackground">
    <bool>true</bool>
   </property>
   <widget class="QGroupBox" name="serialPortBox">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>191</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#serialPortBox
{
border: 1px solid #dddddd;
margin-top:7px;
border-radius:8px;
background-color:  #dddddd;
}
#serialPortBox::title
{
subcontrol-origin:margin;
left:15px;
}</string>
    </property>
    <property name="title">
     <string/>
    </property>
    <widget class="QPushButton" name="connectBtn">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>250</y>
       <width>101</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>  打开</string>
     </property>
     <property name="icon">
      <iconset resource="QtResource.qrc">
       <normaloff>:/images/icon_status_closed.png</normaloff>:/images/icon_status_closed.png</iconset>
     </property>
    </widget>
    <widget class="QLabel" name="dataBitTitle">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>150</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>数据位：</string>
     </property>
    </widget>
    <widget class="QLabel" name="parityTitle">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>110</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>校验位：</string>
     </property>
    </widget>
    <widget class="QLabel" name="portTitle">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>33</y>
       <width>51</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>串口号：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="port">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>30</y>
       <width>101</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="dataBit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>150</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="baudRateTitle">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>波特率：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="baudRate">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="parity">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>110</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="stopBit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>190</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="stopBitTitle">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>停止位：</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="readGroupBox">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>261</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#readGroupBox
{
border: 1px solid #cccccc;
margin-top:7px;
border-radius:8px;
}
#readGroupBox::title
{
subcontrol-origin:margin;
left:15px;
}</string>
    </property>
    <property name="title">
     <string>读取</string>
    </property>
    <widget class="QLabel" name="portTitle_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>43</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>从机地址：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="readDeviceID">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QComboBox" name="readFunc">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="portTitle_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>123</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>寄存器地址：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="readRegister">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QLabel" name="portTitle_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>163</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>寄存器数量：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="readRegisterCount">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>160</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="portTitle_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>83</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>功 能 码：</string>
     </property>
    </widget>
    <widget class="QPushButton" name="readBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>200</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>读取</string>
     </property>
    </widget>
    <widget class="QLabel" name="readResultTitleLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>220</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>读取结果：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="readResultTextView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>241</width>
       <height>71</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Microsoft YaHei UI';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="writeGroupBox">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>20</y>
      <width>261</width>
      <height>321</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#writeGroupBox
{
border: 1px solid #cccccc;
margin-top:7px;
border-radius:8px;
}
#writeGroupBox::title
{
subcontrol-origin:margin;
left:15px;
}</string>
    </property>
    <property name="title">
     <string>写入</string>
    </property>
    <widget class="QLabel" name="portTitle_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>43</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>从机地址：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="writeDeviceID">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>40</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QLabel" name="portTitle_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>83</y>
       <width>61</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>功 能 码：</string>
     </property>
    </widget>
    <widget class="QComboBox" name="writeFunc">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="inputTipLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>241</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>写入值(多个值用逗号隔开，1-on, 0-off)：</string>
     </property>
    </widget>
    <widget class="QTextEdit" name="writeValuesTextView">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>180</y>
       <width>241</width>
       <height>51</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px; font-family:'Microsoft YaHei UI';&quot;&gt;&lt;br /&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="writeBtn">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>275</y>
       <width>80</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>写入</string>
     </property>
    </widget>
    <widget class="QLabel" name="writeResultLabel">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>277</y>
       <width>141</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>写入成功！</string>
     </property>
    </widget>
    <widget class="QLabel" name="portTitle_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>123</y>
       <width>71</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>寄存器地址：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="writeRegister">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>161</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="QtResource.qrc"/>
 </resources>
 <connections/>
</ui>
